packages/frontend/components/Forms/Form.module.scss

```
@@ -0,0 +1,193 @@
+@import 'styles/variables';
+
+$normal-font-size: 24px;
+$grey: #3f3f3f;
+$dark-grey: #373737;
+$error: red;
+
+.form,
+.tabs {
+  max-width: 700px;
+  padding: 0 50px;
+  margin: 0 auto;
+
+  color: $white;
+  font-size: $normal-font-size;
+}
+
+.form.hide {
+  visibility: hidden;
+  position: absolute;
+  left: -100%;
+}
+
+.tabs {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 5px 0.5em;
+  padding-top: 100px;
+
+  label {
+    grid-column: 1 / 4;
+    grid-row: 1;
+  }
+
+  .button {
+    overflow: hidden;
+  }
+  .button.selected {
+    background: $orange;
+  }
+}
+
+.textInput {
+  margin: 25px 0;
+
+  label,
+  input,
+  textarea {
+    font-size: $normal-font-size;
+    color: $white;
+  }
+  label {
+    display: block;
+    margin-bottom: 5px;
+  }
+
+  input {
+    height: 2.3em;
+  }
+
+  textarea {
+    resize: vertical;
+    height: 6em;
+  }
+
+  input,
+  textarea {
+    width: calc(100% - 20px);
+    padding: 0 10px;
+    background: $grey;
+    border: none;
+    border-radius: 10px;
+  }
+
+  input:-webkit-autofill,
+  input:-webkit-autofill:focus,
+  input:-webkit-autofill:active {
+    -webkit-text-fill-color: $white;
+    -webkit-box-shadow: $grey;
+    transition: background-color 99999s ease-in-out 0s;
+  }
+
+  textarea:focus,
+  input:focus {
+    outline: none;
+    box-shadow: 0 0 0 6px $orange;
```

Could you change 6px to 4px? The border looks a bit fat to me. Also make it inset so it doesnt overlap with other components, and the border radius stays consistent.
```suggestion
    box-shadow: inset 0 0 0 4px $orange;
```